README.md

```diff
@@ -1 +1,65 @@
-# LoopMusicSwift
+# LoopMusic
+
+LoopMusic is a music player (iOS app) specially designed for songs that can loop indefinitely, the most prevalent example being video game music.
+
+## At a Glance
+
+### Listen to your favorite songs for as long as you want with seamless looping!
+<p align="center">
+    <img width="350" src="media/main-screen.png" title="LoopMusic's main screen." alt="LoopMusic main screen">
+</p>
+
+### Use the automatic loop finder to locate a seamless loop in just a few button presses!
+<p align="center">
+    <img width="350" src="media/loop-finder.png" title="The Loop Finder screen." alt="Loop Finder screen">
+</p>
+
+## Motivation
+
+Typically, if you want to listen to video game music, you have a few options, each with their own flaws:
+
+1. Listen to a song in-game, either during actual gameplay, or in a [sound test](https://en.wikipedia.org/wiki/Sound_test) mode if one exists. However...
+    - Loading up a game to listen to the music is cumbersome and not very portable. It's even worse if the game doesn't have a sound test mode for easy track selection.
+    - You're limited to songs from a single game at a time.
+    - The audio quality might not be optimal.
+2. If one exists, purchase a game's Official Sound Track (OST) to add to your music library. However...
+    - Most video game tracks are relatively short, typically somewhere between 30 seconds and 2 minutes long. With a fixed-length audio file, you can't listen to a track for an extended period of time like you can in-game (where playback is usually looped indefinitely) without editing the file yourself.
+3. Search for individual songs or compilations that people have uploaded online (for example, on YouTube). However...
+    - The fixed-length nature of individual tracks leads to similar downsides as official OSTs.
+    - There are fairly common practices to circumvent the fixed-length issue, including the use of "extended" tracks that are manually edited to loop for longer periods of time (often 15-30 minutes), livestreams of curated playlists, and long compilations (usually multiple hours in length). But these all share the same flaws:
+        - They are extremely wasteful from a data standpoint
+        - There is only limited customizability for the length and choice of songs
+        - They rely on having internet access, which hinders portability (or require you to download huge audio files, which is not only a waste of memory, but also of questionable legality).
```

Comment:
Is it worth bringing up legality here? If there's no OST and no good way to capture clean audio from recording, you have to do this anyway.

Reply:
True, but I think the more common case is when a track is available through OST or recording, but there's no easy and portable way to listen to it on a seamless loop. In this case the app gives you the option to use the OST/recording. Without the app, you might be forced to do something like downloading an extended audio file from the internet.

Reply:
I see, as a comparison to the other non-app methods. Works for me.